airbyte-integrations/connectors/source-clazar/metadata.yaml

```diff
@@ -0,0 +1,32 @@
+data:
+  allowedHosts:
+    hosts:
+      - api.clazar.io
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: false
```

Comment:
@prashant-mittal9 you probably want to set this to true if you want this connector to work on Airbyte Cloud.